npm/defs.bzl

```diff
@@ -4,33 +4,11 @@
 load(
     "//npm/private:npm_package.bzl",
     _npm_package = "npm_package",
-    _npm_package_lib = "npm_package_lib",
 )
 load(
     "//npm/private:npm_link_package.bzl",
     _npm_link_package = "npm_link_package",
 )
-load(
-    "//npm/private:npm_linked_package_info.bzl",
-    _NpmLinkedPackageInfo = "NpmLinkedPackageInfo",
-)
-load(
-    "//npm/private:npm_package_info.bzl",
-    _NpmPackageInfo = "NpmPackageInfo",
-)
-load(
-    "//npm/private:npm_package_store_info.bzl",
-    _NpmPackageStoreInfo = "NpmPackageStoreInfo",
-)
 
-# export rules & macros as public API
 npm_package = _npm_package
 npm_link_package = _npm_link_package
-
-# export providers as public API
-NpmPackageInfo = _NpmPackageInfo
-NpmPackageStoreInfo = _NpmPackageStoreInfo
-NpmLinkedPackageInfo = _NpmLinkedPackageInfo
```

Comment:
why no longer make the Providers public API?

Reply:
moved to providers.bzl since only rules care about providers and you'll never see them in user BUILD files